doc/developers/contributing.rst

```diff
@@ -14,6 +14,25 @@ adding new algorithms, and the best way to contribute and to help the project
 is to start working on known issues.
 See :ref:`easy_issues` to get started.
 
+.. topic:: **Our community, our values**
+
+    We are a community based on openness and friendly discussions.
+
+    We aspire to treat everybody equally, and value their contributions.
+
+    Decisions are made based on technical merit and consensus.
+
+    Code is not the only way to help the project. Reviewing pull
+    requests, answering questions to help others on mailing lists or
+    issues, organizing and teaching tutorials, working on the website,
+    are all priceless contributions.
+
+    We abide by the principles of openness, respect, and consideration of
+    others of the Python
```

Comment:
There is a line break between "Python" and "Software Foundation" - it was more a style comment ;)